sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/CosmosSchedulers.java

```diff
@@ -16,4 +17,11 @@
         COSMOS_PARALLEL_THREAD_NAME,
         Schedulers.DEFAULT_POOL_SIZE,
         true);
+
+    // Custom bounded elastic scheduler to switch off IO thread to process response.
+    public final static Scheduler TRANSPORT_RESPONSE_BOUNDED_ELASTIC = Schedulers.newBoundedElastic(
```

Comment:
does this need to be released as a resource when the client is closed?

Reply:
I think yes, we need to dispose it when the client gets closed, otherwise it will not let JVM shut down.